bin/common.sh

```diff
@@ -139,7 +147,9 @@ function run() {
   #     modified; please apply your changes to the latest version and try again
   apply_test_resources || apply_test_resources || return $?
 
-  scale_deployment "$(oc get deploy -n perf-test | tail -n 1 | awk '{print $1}')" 10 "${TEST_CASE_NAMESPACE}"
+  if ${SCALE_UP_TEST_DEPLOYMENT}; then
+    scale_deployment "$(oc get deploy -n perf-test | tail -n 1 | awk '{print $1}')" 10 "${TEST_CASE_NAMESPACE}"
+  fi
```

Comment:
I'm not sure about this since it might produce misleading results for e2e latency

Reply:
Like 1 single instance receiving thousands of (big) requests / s

Reply:
10 is for sure a bit arbitrary, but 1 single instance is very small for these tests

Reply:
maybe I would suggest to instead move to define `RECEIVER_DEPLOYMENT_REPLICAS` that would set the actual expected replicas and you can do your experiments

Reply:
Even locally, I can see how it is better to just specify how many replicas are expected, you could still specify
```
export RECEIVER_DEPLOYMENT_REPLICAS=<current-replicas>
```
to avoid waiting

Reply:
Sounds reasonable